python/app/app/main.py

```diff
@@ -4,9 +4,11 @@
 import pickle
 import pandas as pd
 from flask import Flask, jsonify, request
+from flask_cors import CORS
```

Comment:
This needs to be added to pyproject.toml, then run `poetry lock`, and `poetry export -f requirements.txt > requirements.txt`